Makefile

```diff
@@ -0,0 +1,17 @@
+
+.PHONY: depinstall clean sdist test
```

Comment:
The `README` doesn't correctly reference actual targets in the `Makefile`

Should also have a sensible `make` default. Normal behaviour should be along the lines of:
```
$ make
$ make test
$ make install
```

The `make (all)` should pull in any dependencies etc that it needs, the `test` should, well, run the tests, and the more optional `install` should potentially `pip` install the lib to the local env.

https://www.gnu.org/prep/standards/html_node/Standard-Targets.html